packages/esm-patient-search-app/src/compact-patient-search/compact-patient-search.resource.ts

```diff
@@ -0,0 +1,34 @@
+import { type FHIRIdentifier, type FHIRPatientType, type SearchedPatient } from '../types';
+
+export function toFhirPatient(patient: SearchedPatient): FHIRPatientType {
+  const preferredAddress = patient.person.addresses?.find((address) => address.preferred);
+  return {
+    id: patient.uuid,
+    name: [
+      {
+        id: String(Math.random()), // not used
+        given: [patient.person.personName.givenName, patient.person.personName.middleName],
+        family: patient.person.personName.familyName,
+        text: patient.person.personName.display,
+      },
+    ],
+    gender: patient.person.gender,
+    birthDate: patient.person.birthdate,
+    deceasedDateTime: patient.person.deathDate,
+    deceasedBoolean: patient.person.dead,
+    identifier: patient.identifiers as unknown as Array<FHIRIdentifier>,
+    address: preferredAddress
+      ? [
+          {
+            id: String(Math.random()), // not used
+            city: preferredAddress.cityVillage,
+            country: preferredAddress.country,
+            postalCode: preferredAddress.postalCode,
+            state: preferredAddress.stateProvince,
+            use: 'home',
+          },
+        ]
+      : [],
+    telecom: patient.attributes?.filter((attribute) => attribute.attributeType.display == 'Telephone Number'),
+  };
+}
```

Comment:
Hi @chibongho!
This particular change needs to be a part of the whole change in the patient search app to migrate the search API from OpenMRS REST to FHIR.
For now, we need to pass the FHIR object inside the banner actions, which is what we get from the `usePatient` hook.
Apologies.
Created a ticket here: https://openmrs.atlassian.net/browse/O3-4079